js/ui/file_manager/ui.file_manager.js

```diff
@@ -258,6 +258,7 @@ class FileManager extends Widget {
         this._disposeWidget(this._itemView);
 
         this._createItemView(this._$itemsPanel, viewMode);
+        this.option("itemView.mode", viewMode);
```

Comment:
Without additional code modifications this approach leads to one additional excessive` repaint()` method call. We should remove this excessive call in that case.